### ISSUE TITLE
Azure Batch - Get Node Agent Sku Formatting Error

### ISSUE THREAD

**Issue by bgklein** [Azure/azure-powershell#7863]
### Description
Formatting error for Get-AzureBatchNodeAgentSku cmdlet

### Script/Steps for Reproduction
```powershell
$Context = Get-AzureRmBatchAccountKeys -AccountName "personal"
Get-AzureBatchNodeAgentSku -BatchContext $Context
```
Output:
![image](https://user-images.githubusercontent.com/18502617/48726390-08f2c980-ebe4-11e8-9581-fe68e4dee603.png)

Actual values stored:
![image](https://user-images.githubusercontent.com/18502617/48726401-1445f500-ebe4-11e8-9fdc-80fdc39b836e.png)

### Module Version
N/A

### Environment Data
N/A

### Debug Output
N/A

@matthchr issue we discussed for tracking

**Comment by matthchr** [comment]
Thanks @bgklein - it looks like this is a display issue with the table formatting. I'll take a look at fixing it.
@cormacpayne can you assign this to me and tag it as "batch"?

**Comment by matthchr** [comment]
This is fixed in `1.0.1-preview` of the `Az.Batch` module.

You can acquire it by running the following:
`Install-Module -Name Az.Batch -Repository PSGallery -AllowPrerelease -Force`
